### PR TITLE
console: Added hint to user entries

### LINKED ISSUE #3429: Add `(This is you)` hint to user entries in user-management tables
#### Summary

Tables that show user entries should append a `(This is you)` message next to the user id, to improve clarity.

See https://github.com/TheThingsNetwork/lorawan-stack/issues/1563

#### Why do we need this?

Improve clarity for the user.

#### What is already there? What do you see now?

Plain display of the user id.

#### What is missing? What do you want to see?

A hint that an entry refers to the current user when navigating to `/console/admin/user-management`.

#### Environment

v3.10

#### How do you propose to implement this?

See https://github.com/TheThingsNetwork/lorawan-stack/pull/3407

#### How do you propose to test this?

Manual testing

#### Can you do this yourself and submit a Pull Request?

@asmulko

### PR DESCRIPTION
**Pull request by asmulko**
#### Summary
Closes #3429 

#### Changes

- Added hint to user entries in user-management table.
- Updated collaborators table to use shared message (This is you)

Previously:

Now:

#### Testing

- Manually tested

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/console/containers/users-table/index.js
Comment (pgalic96) [inline pkg/webui/console/containers/users-table/index.js]: I think it's better to avoid using spread operators for props, especially if it initially works and we defined all props that are supplied to the component. Is this right @kschiffer ?

**Review by pgalic96 — APPROVED** [review]
Looks good, please use imperative verb for commit (Add instead of added). Check contributing guidelines on commit messages: https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.10/CONTRIBUTING.md

Also it is fine to just use the number without `#` for the branch name.

Also see one review comment and maybe wait for a review by @bafonins or @kschiffer as well.

File: pkg/webui/console/containers/users-table/index.js
Comment (kschiffer) [reply pkg/webui/console/containers/users-table/index.js]:
```suggestion
```

It depends. @asmulko, you likely added this because we do this in other table components as well. I was more lax regarding this until experiencing some issues. When spreading props, it's important to make sure that no unrelated props are being passed down, since that can lead to nasty bugs. To easily avoid that, it's usually better to be explicit about what props to pass (as in not using spread). In this particular case, there's also no use in passing the props.

**Review by kschiffer — APPROVED** [review]
LGTM, see my comment about removing the spread.

**Comment by asmulko** [comment]
Thanks for the comments, fixed CI and removed unnecessary props

**Pull request merged**
